v2_task_test.go

```diff
@@ -23,21 +23,21 @@ func TestTask_Read(t *testing.T) {
 	"time_created": "2019-06-18T00:00:00",
 	"time_started": "2019-07-18T00:00:00",
 	"time_ended": "2019-08-18T00:00:00",
-	"duration": 4294967295
+	"duration": "4294967295"
```

Comment:
I don't think Archivematica ever writes the duration in seconds as a string right?

I'm looking at https://github.com/artefactual/archivematica/blob/2c03a79e710d655c3740ddbc4bc62eebd4e4b4fe/src/dashboard/src/components/helpers.py#L163-L170:

It sounds like these are all the potential values we'll see:
1. Signed integer: `1`, `0`, `-1`, ...
2. Empty string: `""`
3. Minus one: `"< 1"`

Reply:
I looked into and it is a bit weird that it can return a negative number as a possibility but nonetheless it does. I do think that we should actually just have a panic or something if we do get a negative number because that seems like something went wrong.

Reply:
what a wacky set of returns